lib/ga-tag.js

```diff
@@ -39,22 +49,28 @@ const setupPageMap = routes => {
   return pageMap;
 };
 
-module.exports = (app, config) => {
+  module.exports = (app, config) => {
   const gaTagId = config.gaTagId;
   const ga4TagId = config.ga4TagId;
+  const gtmTagId = config.gtmTagId;
+  const appName = config.appName;
   const gaCrossDomainTrackingTagId = config.gaCrossDomainTrackingTagId;
   const routes = config.routes;
 
   if (gaTagId || ga4TagId) {
     const pageMap = setupPageMap(routes);
 
     app.use((req, res, next) => {
+      const page = pageView(req.path, pageMap);
       res.locals.gaAllowDebug = config.env === 'development';
       res.locals.gaTagId = gaTagId;
       res.locals.ga4TagId = ga4TagId;
+      res.locals.gtmTagId = gtmTagId
+      res.locals.appName = appName
```

Comment:
why do you need the appName?

Reply:
I was using it before, but no longer need it. Good spot